### ISSUE TITLE
Priority 2 Accessibility issue: Colour picker status update

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6739]
**What**
The hex code updates upon selection of a colour with either the buttons or the slider, but this is not announced to screen reader users. Please also note, not all users will be familiar with HEX codes.

Solution:
An aria role of ‘status’ on an element containing a dynamically updating message will ensure that screen reader users are informed when the content within the element changes, without needing to move their focus or search the page for the message. For more information, please refer to F103: Failure of Success Criterion 4.1.3 due to providing status messages that cannot be programmatically determined through role or properties.
So that the purpose of this information is clear, some additional text may also be helpful. For example, ‘Colour Red Selected: Hex code #E02020’.

**Where**

 https://wellcomecollection.org/images?query=train

**Comment by gestchild** [comment]
N.B I've changed this to use the colour name for the 10 predefined colour searches we have, but it's not really possible to give useful names to all the possible hex values
